src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
+
+  type Key = Array[Byte]
+  type Value = Array[Byte]
+
+  /**
+    * This function obtains the associated value to a key, if there exists one.
+    *
+    * @param namespace which will be searched for the key.
+    * @param key
+    * @return the value associated with the passed key.
+    */
+  def get(namespace: Byte, key: Key): Option[Value]
```

Comment:
I would avoid adding one more place which potentially could throw NPEs. We have `Option.getOrElse` and `Option.get` for such scenarios.